app/src/main/java/com/simpragma/recipe/roomDatabase/RecipeDao.java

```diff
@@ -0,0 +1,27 @@
+package com.simpragma.recipe.roomDatabase;
+
+import android.arch.persistence.room.Dao;
+import android.arch.persistence.room.Insert;
+import android.arch.persistence.room.Query;
+import android.util.Log;
+
+import java.util.List;
+
+/**
+ * Created by Madvesha on 2/12/18.
+ */
+@Dao
+public interface RecipeDao {
+
+    //Retrieving all the data from RoomDB
+    @Query("SELECT * FROM reciperoomdb")
```

Comment:
Why the database is named `reciperoomdb`?

Reply:
I given table name recipetable

Reply:
Table name is recipetable